pyproject.toml

```diff
@@ -197,7 +197,7 @@ build-backend = "poetry.core.masonry.api"
 
 # https://beta.ruff.rs/docs/configuration/#using-pyprojecttoml
 [tool.ruff]
-select = ["E", "F", "W", "PIE", "C4", "ISC", "RUF100"]
+select = ["E", "F", "W", "PIE", "C4", "ISC", "RUF100", "B006"]
```

Comment:
think we want all the B's

Reply:
Added all the B's except a few I marked below